### DISCUSSION TITLE
Open shell 4.4.190 and start menu in Windows 11

### DISCUSSION THREAD

**Discussion by valerio-vanni** [Open-Shell/Open-Shell-Menu#1669] [General]
In Windows 11 and 4.4.170 version, I have in central position Windows 11 start menu, and on the left corner Open shell menu.
And I find it ok: I have both menu available with a single click on screen.

Upgrading to 4.4.190, Open shell menu goes to central position and replaces Windows 11 start menu.

Two drawbacks:
1) Reaching  Windows 11 menu requires 1 more click to open (since you have to click open shell one before)
2) For some people, not finding anything on the left corner can be confusing. Yes, it's the same with a plain Windows 11 install, but an advantage of open shell is gone... (finding old menu in the same place)

Is there some setting on 4.4.190 that can trigger previous behavior?

**Comment by ge0rdi** [comment]
Please, see #1252 regarding Start button positioning.

> Reaching Windows 11 menu requires 1 more click to open

You can use Shift+click on Start button (or Shift+Win) to get original Windows Start menu.

**Reply by xenos-github** [reply]
No, no, no! This is about position, not accessibility. Users are telling you they want your button in the lower left hand lower corner. Please listen.

**Reply by ge0rdi** [reply]
Users can enable _left_ taskbar alignment and will have everything where it used to be.

**Comment by johnwasme** [comment]
How do you enable the alignment?In open shell it has a setting to enable alignment to the corner of the screen but doesn't do anything,just stays in the middle.

**Reply by ge0rdi** [reply]
`Open-Shell` start button/menu is always aligned with original windows start button (as its purpose is to replace it).
Thus it depends on taskbar alignment whether start button/menu is on the left or in the middle.

**Comment by xenos-github** [comment]
It's in windows' taskbar settings.

**Comment by valerio-vanni** [comment]
> `Open-Shell` start button/menu is always aligned with original windows start button (as its purpose is to replace it). Thus it depends on taskbar alignment whether start button/menu is on the left or in the middle.

This way seems much better.
The separate menus were the top, but at least I can have a consistent interface ("start menu is on the low left corner") between Windows 10 and Windows 11 with open shell installed.

**Comment by f14tomcat2** [comment]
> `Open-Shell` start button/menu is always aligned with original windows start button (as its purpose is to replace it). Thus it depends on taskbar alignment whether start button/menu is on the left or in the middle.

The Windows Start Menu is in essence a large Metro style display of apps, whereas Open Shell is a convenient common sense interface to user apps and system functions.  As MS continues to take the Start Menu further and further off into left field, Open Shell remains stable and predictable.  I know where I put things.  To me, and I would imagine many others, Open Shell and the Windows Start Menu are 2 different worlds coexisting in one OS.  Versions .180 and prior had a convenient separate of function.  If it could be done there why can't it be optional for .190 and greater?  Pic of .180.  This is convenient and functional.

<img width="526" alt="2023-08-23_15h24_08" src="https://github.com/Open-Shell/Open-Shell-Menu/assets/142270598/0f6182e0-ee1f-428e-94bf-79ebc9a68d23">
